### ISSUE TITLE
influxdb : character '_' in tag $group

### ISSUE THREAD

**Issue by amic81** [sitespeedio/sitespeed.io#2011]
Hi,

because in influxdb in the $group tag instead of the character '.' did you use the character '_'?

Example

"www_sitespeed_io" instead of "www.sitespeed.io"

Thanks
amic81

**Comment by soulgalore** [comment]
Yep, it's because we use the same way to creating the keys/tags as in Graphite and dots in Graphite means a new part of the key.

Best
Peter

**Comment by amic81** [comment]
ok, I understand.

Thanks
amic81

**Comment by amic81** [comment]
Hello,

it would be possible to overwrite (from the outside) the regular expression

```
function toSafeKey(key) {
  return key.replace(/[.~ /+|,:?&%]|%7C/g, '_');
}
```

so that people like me do not pass from Graphite but only use influxdb can save $group with the character **'.'** and not **'_'**?

Thank you
amic81

**Comment by soulgalore** [comment]
No but I think we can do it configurable, so we keep it as default and then by config you can change to a dot? Do you have time to do a PR, I can guide you?

Best
Peter

**Comment by amic81** [comment]
Hi, that's great. I have time to manage a PR.
Guide me as well.

amic81

**Comment by amic81** [comment]
Hi,
I made the requested changes, currently in my local branch.

I would like to be able to push this branch so that I can request the PR ... or I have to open a fork of the project (https://help.github.com/articles/creating-a-pull-request-from-a-fork/) and then request the PR later?

Let me know how you prefer it to proceed.

Thank you
amic81

**Comment by soulgalore** [comment]
Cool, I think the easiest way is to do it like this: Fork the project, create a branch in your fork, add your changes there, push your branch to Github, and then you can use the Github gui to make the PR to sitespeed.io

**Comment by amic81** [comment]
Done.

I hope I have done everything correctly. ;-)

**Comment by soulgalore** [comment]
Released in 7.0
